modules/log6x-configuring-otlp-output.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-configuring-otlp-output_{context}"]
+= Configuring OTLP output
+
+The OpenTelemetry Protocol (OTLP) output allows the cluster administrator to collect and forward logs to OTLP receivers. The OTLP output uses the protocol defined by the https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry Observability framework] to send data over HTTP with JSON encoding.
+
+:FeatureName: The OTLP output log forwarder
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+* Create or edit a `ClusterLogForwarder` CR to enable forwarding using OTLP by adding the following annotation:
++
+.Example ClusterLogForwarder CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  annotations:
+    observability.openshift.io/tech-preview-otlp-output: "enabled" # <1>
+  name: clf-otlp
+spec:
+  serviceAccount:
+    name: <service_account_name> 
+  outputs:
+  - name: otlp
+    type: otlp
+    otlp:
+      tuning:
+        compression: gzip 
+        delivery: atLeastOnce
```

Comment:
This should be `deliveryMode: AtLeastOnce`

Reference: https://github.com/openshift/cluster-logging-operator/blob/release-6.0/bundle/manifests/observability.openshift.io_clusterlogforwarders.yaml#L2022